runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/environment/DockerCommand.java

```diff
@@ -61,8 +61,12 @@ private DockerCommand(String dockerExecutable, Duration commandTimeout) {
   /**
    * Runs the given container image with the given command line arguments. Returns the running
    * container id.
+   *
+   * @param imageTag the name of the image to run
+   * @param dockerOpts options to provide to docker
+   * @param args arguments to provide to the container
    */
-  public String runImage(String imageTag, List<String> args)
```

Comment:
Apparently we already had this in the hacking branch: https://github.com/bsidhom/beam/blob/3e9b24a1247033e5d75f80e106d3d75052389799/runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/environment/DockerWrapper.java#L61.

We hadn't triaged/migrated this part yet. I'll update it in the sheet: https://docs.google.com/spreadsheets/d/1KF5n5eTq0neIqwhIkFbvRTbp0G5mqmJ9O4ywSMWtfq0/edit#gid=1955143518.